lib/reference/keywordSets.js

```diff
@@ -277,10 +274,21 @@ keywordSets.otherPseudoClasses = new Set([
 	'target',
 	'user-error',
```

Comment:
not sure if it should be removed or not
see https://lists.w3.org/Archives/Public/www-style/2015Sep/0111.html

Reply:
If it's been dropped, let's remove it too.